content-src/components/Base/Base.jsx

```diff
@@ -86,8 +88,10 @@ export class _Base extends React.PureComponent {
       if (window.location.hash.startsWith("#asrouter") ||
           window.location.hash.startsWith("#devtools")) {
         return (<ASRouterAdmin />);
+      } else if (!didLogDevtoolsHelpText) {
```

Comment:
I don't know if you need this to be a `else if`, just a new `if`?

Reply:
I guess all the `else if` means is if you enable it while already on the devtools url, you don't get a log? That probably doesn't matter much, if I'm reading this right?

Reply:
I get this now, makes sense. If you're one one of the above urls, logging how to find those urls would be pointless.